deploy/2-configure-feeds.ts

```diff
@@ -26,70 +26,86 @@ interface Oracle {
   oracles: [string, string, string];
   enableFlagsForOracles: [boolean, boolean, boolean];
   underlyingOracle: Contract;
-  getTokenConfig: (asset: Asset, networkName: string) => void;
+  getTokenConfig?: (asset: Asset, networkName: string) => void;
 }
 
 interface Oracles {
   [key: string]: Oracle;
 }
 
-const pythID: Config = {
-  bsctestnet: {
-    BNX: "0x843b251236e67259c6c82145bd68fb198c23e7cba5e26c995e39d8257fbf8eb8",
-    WBTC: "0xf9c0172ba10dfa4d19088d94f5bf61d3b54d5bd7483a322a982e1373ee8ea31b",
-    WBNB: "0xecf553770d9b10965f8fb64771e93f5690a182edc32be4a3236e0caaa6e0581a",
-    AUTO: "0xd954e9a88c7f97b4645b535869aba8a1e50697270a0afb09891accc031f03880",
-    CAKE: "0x3ea98c0336f6a8506dc34567da82178f6f7a664b206ae8eaf8ab8d96721ecccc",
-  },
-};
-
 const chainlinkFeed: Config = {
   bsctestnet: {
     BNX: "0xf51492DeD1308Da8195C3bfcCF4a7c70fDbF9daE",
-    WBTC: "0x5741306c21795FdCBb9b265Ea0255F499DFe515C",
-    BNB: "0x5741306c21795FdCBb9b265Ea0255F499DFe515C",
-    CAKE: "0x81faeDDfeBc2F8Ac524327d70Cf913001732224C",
-    BUSD: "0x9331b55D9830EF609A2aBCfAc0FBCE050A52fdEa",
-    WBNB: "0x2514895c72f50D8bd4B4F9b1110F0D6bD2c97526",
+    BTCB: "0x5741306c21795FdCBb9b265Ea0255F499DFe515C",
   },
 };
 
 const assets: Assets = {
```

Comment:
Do we need a config entry for VAI?

Reply:
No. VAI address is configured during deployed as part of deploy script.

Reply:
Agreement offline, we'll provide a different script to configure the "core markets", including VAI